lib/isMobilePhone.js

```diff
@@ -50,7 +50,9 @@ var phones = {
   'en-US': /^((\+1|1)?( |-)?)?(\([2-9][0-9]{2}\)|[2-9][0-9]{2})( |-)?([2-9][0-9]{2}( |-)?[0-9]{4})$/,
   'en-ZA': /^(\+?27|0)\d{9}$/,
   'en-ZM': /^(\+?26)?09[567]\d{7}$/,
+  'es-CO': /^(\+?57)?([1-8]{1}|3[0-9]{2})?[2-9]{1}\d{6}$/,
   'es-CL': /^(\+?56|0)[2-9]\d{1}\d{7}$/,
+  'es-EC': /^(\+?593|0)([2-7]|9[2-9])\d{7}$/,
```

Comment:
Did you add both `es-CO` and `es-EC`?

Reply:
If so, add tests for both.

Reply:
Didn't check 'es-EC'. Seems like a spacing change.

Reply:
Actually, the es-EC was already introduced in src/lib/isMobilePhone.js. Perhaps checking with @mrcrow85?